raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -53,11 +52,11 @@ library ChannelManagerLibrary {
         return (0x0, false, 0, 0);
     }
 
-    /// @notice newChannel(address, uint) to create a new payment channel between two parties
+    /// @notice Create a new payment channel between two parties
     /// @dev Create a new channel between two parties
-    /// @param partner (address) the address of the partner
-    /// @param settle_timeout (uint) the settle timeout in blocks
-    /// @return (address) the address of the NettingChannelContract.
+    /// @param partner the address of the partner
+    /// @param settle_timeout the settle timeout in blocks
+    /// @return the address of the NettingChannelContract.
```

Comment:
Yes I think we should. Also helps differentiate where the contents of the docstring start.